src/benchmarks/micro/libraries/System.Security.Cryptography/Perf.RandomNumberGenerator.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Security.Cryptography.Tests
+{
+    [BenchmarkCategory(Categories.Libraries, Categories.NoWASM)]
+    public class Perf_RandomNumberGenerator
+    {
+        [Benchmark]
+        public byte[] GetBytes()
+        {
+            using RandomNumberGenerator rng = RandomNumberGenerator.Create();
+            byte[] buffer = new byte[8];
```

Comment:
this benchmark measures the cost of creating `RandomNumberGenerator` and a byte array, the call to `rng.GetBytes` and the disposal of `RandomNumberGenerator`. Perhaps it would be better to move the initialization to setup method, so it focuses only on `GetBytes`?

Reply:
Done